.claude/commands/estimate.md

```
@@ -0,0 +1,40 @@
+**Purpose**: Project complexity and time estimation
+
+---
+
+@include shared/universal-constants.yml#Universal_Legend
+
+## Command Execution
+Execute: immediate. --plan→show plan first
+Legend: Generated based on symbols used in command
+Purpose: "[Action][Subject] in $ARGUMENTS"
```

markdownlint-cli2 (0.17.2)

Reference links and images should use a label that is defined
Missing link or image reference definition: "subject"

(MD052, reference-links-images)